ScriptHookVDotNet.vcxproj

```diff
@@ -138,35 +138,11 @@
   </ItemGroup>
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.targets" />
   <!-- Compile C# source files -->
-  <Target
-    Name="CsCompile"
-    BeforeTargets="ClCompile"
-    Inputs="@(CsCompile)"
-    Outputs="$(IntDir)$(TargetName).netmodule">
-    <Csc
-      Sources="@(CsCompile)"
-      AllowUnsafeBlocks="%(CsCompile.AllowUnsafeBlocks)"
-      DebugType="%(CsCompile.DebugType)"
-      DisabledWarnings="%(CsCompile.NoWarn)"
-      DocumentationFile="$(IntDir)$(TargetName).xdc"
-      EmitDebugInformation="%(CsCompile.DebugSymbols)"
-      LangVersion="%(CsCompile.LangVersion)"
-      Optimize="%(CsCompile.Optimize)"
-      OutputAssembly="$(IntDir)$(TargetName).netmodule"
-      PdbFile="$(IntDir)$(TargetName).netmodule.pdb"
-      TargetType="Module"
-      TreatWarningsAsErrors="%(CsCompile.TreatWarningsAsErrors)"
-      Utf8Output="%(CsCompile.Utf8Output)"
-      WarningLevel="%(CsCompile.WarningLevel)"
-      WarningsAsErrors="%(CsCompile.WarningsAsErrors)"
-      WarningsNotAsErrors="%(CsCompile.WarningsNotAsErrors)"
-      ToolExe="$(CscToolExe)"
-      ToolPath="$(CscToolPath)" />
+  <Target Name="CsCompile" BeforeTargets="ClCompile" Inputs="@(CsCompile)" Outputs="$(IntDir)$(TargetName).netmodule">
+    <Csc Sources="@(CsCompile)" AllowUnsafeBlocks="%(CsCompile.AllowUnsafeBlocks)" DebugType="%(CsCompile.DebugType)" DisabledWarnings="%(CsCompile.NoWarn)" DocumentationFile="$(IntDir)$(TargetName).xdc" EmitDebugInformation="%(CsCompile.DebugSymbols)" LangVersion="%(CsCompile.LangVersion)" Optimize="%(CsCompile.Optimize)" OutputAssembly="$(IntDir)$(TargetName).netmodule" PdbFile="$(IntDir)$(TargetName).netmodule.pdb" TargetType="Module" TreatWarningsAsErrors="%(CsCompile.TreatWarningsAsErrors)" Utf8Output="%(CsCompile.Utf8Output)" WarningLevel="%(CsCompile.WarningLevel)" WarningsAsErrors="%(CsCompile.WarningsAsErrors)" WarningsNotAsErrors="%(CsCompile.WarningsNotAsErrors)" ToolExe="$(CscToolExe)" ToolPath="$(CscToolPath)" />
```

Comment:
Please just revert this formatting change in the project file (which Visual Studio applied when you added the new file), and this is good to go.